File: telerik_ui_for_winforms_courseware_chm/Courseware/Grid/VB/RadGridView/HierarchyDesigner/RadGridViewLab5.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms

Namespace _05_HierarchyDesigner
	Public Partial Class RadGridViewLab5
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub RadGridViewLab5_Load(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			' TODO: This line of code loads data into the 'adventureWorksDataSet1.EmployeeDepartmentHistory' table. You can move, or remove it, as needed.
			Me.employeeDepartmentHistoryTableAdapter.Fill(Me.adventureWorksDataSet1.EmployeeDepartmentHistory)
			' TODO: This line of code loads data into the 'adventureWorksDataSet1.Department' table. You can move, or remove it, as needed.
			Me.departmentTableAdapter.Fill(Me.adventureWorksDataSet1.Department)

		End Sub
	End Class
End Namespace

File: telerik_ui_for_winforms_courseware_chm/Courseware/Grid/CS/RadGridView/HierarchyDesigner/RadGridViewLab5.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace _05_HierarchyDesigner
{
    public partial class RadGridViewLab5 : Form
    {
        public RadGridViewLab5()
        {
            InitializeComponent();
        }

        private void RadGridViewLab5_Load(object sender, EventArgs e)
        {
            // TODO: This line of code loads data into the 'adventureWorksDataSet1.EmployeeDepartmentHistory' table. You can move, or remove it, as needed.
            this.employeeDepartmentHistoryTableAdapter.Fill(this.adventureWorksDataSet1.EmployeeDepartmentHistory);
            // TODO: This line of code loads data into the 'adventureWorksDataSet1.Department' table. You can move, or remove it, as needed.
            this.departmentTableAdapter.Fill(this.adventureWorksDataSet1.Department);

        }
    }
}
